### PR TITLE
removed redundant bits of code

### PR DESCRIPTION
**Pull request by FullStackForger**
Favouring KISS principle, there is no need for `node ./node_modules/.bin/` in `package.json`.  
And here is why:
1. Scripts configured in `package.json` auto execute binaries from `./node_modules/.bin`
2. `./node_modules/.bin/istanbul` is an executable node script, no need to prefix it with `node`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Learn how to use istanbul for tracking test/code coverage in your JS projects",
   "main": "test.js",
   "scripts": {
-    "coverage": "node ./node_modules/.bin/istanbul cover ./test.js",
+    "coverage": "istanbul cover ./test.js",
```
Comment (nelsonic) [inline package.json:7]: @indieforger agree that this is simpler. 
the _reason_ we used the full path was so people are not confused as to the location of the Istanbul script and don't end up installing it _globally_...

**Comment by nelsonic** [comment]
merging. :+1: 
thanks.

File: package.json (hunk 1 above)
Comment (FullStackForger) [inline package.json:7]: Good point. However installing it globally is not the bad idea for its awesome CLI. 
We can actually mention it in docs too. I am happy to do another PR. Does it sound good?

Comment (nelsonic) [inline package.json:7]: @indieforger yeah, Istanbul is a good tool. We prefer to have our `devDependencies` explicit in the `package.json` though and for it to be _clear_ which version people are using for the project they are working on. 
That way we never have the situation where someone is using a super old or "_alpha_" version (_globally_) which is incompatible with the project they are working on... We've had that before in the past and took ages to figure out.
